lib/lotus/model/adapters/abstract.rb

```diff
@@ -162,6 +170,15 @@ def command(query)
         def query(collection, &blk)
           raise NotImplementedError
         end
+
+        # Create a transaction
+        #
+        # @param options [Hash] options for transaction
+        #
+        # @since x.x.x
+        def transaction(options = {})
+          raise MethodNotSupported
```

Comment:
remove this Exception and only yield? and document that only available in sql adapter?